mitmproxy/options.py

```diff
@@ -70,6 +70,7 @@ def __init__(
             mode = "regular",  # type: str
             no_upstream_cert = False,  # type: bool
             rawtcp = False,  # type: bool
+            websockets=False,  # type: bool
```

Comment:
I think we should enable it - so people can start using it right with the next release?
Or do you think it might be buggy?